src/test-run/index.js

```diff
@@ -490,7 +494,7 @@ export default class TestRun extends AsyncEventEmitter {
         if (isAsyncExpression)
             expression = `(async () => { return ${expression}; }).apply(this);`;
 
-        const result = this._evaluate(expression);
+        const result = executeJsExpression(expression, this, { skipVisibilityCheck: false });
```

Comment:
The `_evaluate` method has the `try/catch` block inside of it. Don't we need it anymore?
If so, can we remove the `_evaluate` method? As I can see, we use it only in your case and in the one test.